sdk/core/core-tracing/README.md

```diff
@@ -42,24 +39,16 @@ Some incompatibility between the libraries is due to mismatches between the Open
 | 1.0.0-preview.11 | 2.1.0-2.1.3          | 1.0.0-rc.0         |
 | 1.0.0-preview.12 | ^2.1.4               | ^1.0.0             |
 | 1.0.0-preview.13 | ^2.1.4               | ^1.0.0             |
+| 1.0.0-preview.14 | ^2.2.3               | ^1.0.0             |
+| 1.0.0            | ^2.2.3               | ^1.0.0             |
 
 Please see the [troubleshooting](#troubleshooting) section for additional information about handling compatibility errors.
 
 ## Examples
 
-### Enabling tracing using OpenTelemetry
-
-```ts
-const opentelemetry = require("@opentelemetry/api");
-const { NodeTracerProvider } = require("@opentelemetry/node");
-const { SimpleSpanProcessor, ConsoleSpanExporter } = require("@opentelemetry/tracing");
+### Using core-tracing as a package owner
 
-const provider = new NodeTracerProvider();
-provider.addSpanProcessor(new SimpleSpanProcessor(new ConsoleSpanExporter()));
-provider.register();
-
-// Call some client library methods using automatic span propagation.
-```
+Examples can be found in the `samples` folder.
 
 ### Manual Span Propagation using OpenTelemetry
```

Comment:
is the following example still relevant? It looks outdated.

Reply:
So, I was debating whether this belongs here or in the OpenTelemetry package readme. It would be good to point out that `tracingContext` can still be passed in for manual propagation, and it's defined in this package, but this is also OTel specific.

Do you think I should move it to the OTel instrumentation package instead?

Reply:
I think so, since the top of this README shoos away SDK consumers, whereas if they want to use tracing they'll probably read the plugin's README

Reply:
Yeah, makes sense. I moved this section over to the plugin readme 👍